src/v2/api/teacher/DisciplineTeacherService.ts

```diff
@@ -125,7 +131,13 @@ export class DisciplineTeacherService {
   }
 
   async getUniqueQuestions (id: string) {
-    const { disciplineTeachers } = await this.disciplineTeacherRepository.getDiscipline(id);
+    const { disciplineTeachers } = await this.disciplineRepository.findBy({
```

Comment:
isn't better to call this.disciplineTeacheRepository.findMany({
  where: {
    discipline: {
      id,
    },
  },
});

check other implementations of findMany method.